cloudevents/sdk/event/v1.py

```diff
@@ -11,10 +11,15 @@
 #    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
 #    License for the specific language governing permissions and limitations
 #    under the License.
+from __future__ import annotations
+
 import typing
 
 from cloudevents.sdk.event import base, opt
 
+if typing.TYPE_CHECKING:
+    from typing_extensions import Self
```

Comment:
Self is not in stdlib `typing` module in older Python versions, so needs a typing-only import.